Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,67 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        public static int FindDiv(String number, String result)  {

```

Comment:
You can use full names also. Does `GetDivision` seem more appropriate instead of `FindDiv`?